Bio/SeqIO/__init__.py

```diff
@@ -647,12 +647,6 @@ def parse(handle, format, alphabet=None):
     # string mode (see the leading r before the opening quote).
     from Bio import AlignIO
 
-    # Hack for SFF, will need to make this more general in future
```

Comment:
Does this mean we can remove ``_BinaryFormats`` now, or is it still used in other tests?

Reply:
It is still being used elsewhere. We can remove _BinaryFormats later.